packages/vaadin-charts/vaadin-chart.js

```diff
@@ -1,3 +1,2 @@
-import './vaadin-chart-default-theme.js';
-import './src/vaadin-chart.js';
-export * from './src/vaadin-chart.js';
```

Comment:
This module defines the legacy class alias `ChartElement`, removing the export breaks the smoke tests here: https://github.com/vaadin/web-components/runs/4166037848?check_suite_focus=true.

The file should be aligned with the other packages, which should look something like this:
```
import './theme/lumo/vaadin-chart.js';
export * from './src/vaadin-chart.js';
```

Reply:
Done.